unity-game/Assets/Scripts/ElevatorScripts/ManualElevator.cs

```diff
@@ -0,0 +1,226 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ManualElevator : MonoBehaviour {
```

Comment:
"Extend" is [inheritance](http://csharp.net-tutorials.com/classes/inheritance/) in object-oriented programming. You will end up with 3 scripts `MovingPlatform`, `ManualElevator`, `AutoElevator` where the second two are subclasses of `MovingPlatform`